contracts/Property.sol

```diff
@@ -1,26 +1,19 @@
 pragma solidity ^0.5.0;
 
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20Detailed.sol";
 import "./UseState.sol";
 import "./Allocator.sol";
 
-contract Property is ERC20, ERC20Detailed, UseState {
-	address public _owner;
-
+contract Property is ERC20, Ownable, ERC20Detailed, UseState {
 	constructor(
-		address _own,
 		string memory _name,
 		string memory _symbol,
 		uint8 _decimals,
 		uint256 _supply
-	) public ERC20Detailed(_name, _symbol, _decimals) {
```

Comment:
In here, `msg.sender` is Property Factory Contract, so I think required passing `address _own`.